go-controller/pkg/ovn/master.go

```diff
@@ -790,10 +789,14 @@ func (oc *Controller) ensureNodeLogicalNetwork(node *kapi.Node, hostSubnets []*n
 	return oc.lsManager.AddNode(nodeName, logicalSwitch.UUID, hostSubnets)
 }
 
-func (oc *Controller) addNodeAnnotations(node *kapi.Node, hostSubnets []*net.IPNet) error {
-	gwLRPIPs, err := oc.joinSwIPManager.EnsureJoinLRPIPs(node.Name)
+func isError(err error) bool {
```

Comment:
since this is just used once and pretty generic, just declare it inside of the retry function calling it